setup.py

```diff
@@ -28,7 +28,8 @@
         "nose>=1.3.4",           # ensure in virtualenv
         "fedmsg==0.8.0",
         "mock==1.0.1",
-        ],
+        "Django==1.7.1",
```

Comment:
In theory it's vaguely sad for `setup.py` to pin to particular versions, but I think for now that's a pretty reasonable thing to do.

It has the upside of making sure that if people decide to install the app via `setup.py` they have a chance of having it actually work.

On the other hand, we want to eventually phase out supporting installing the app via `setup.py` and instead require people to use the packages in Debian for these Python modules.

But I think it's nice to keep this metadata here. But I could probably be convinced that it's needless and dangerous duplication.